compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -5478,10 +5479,14 @@ private BLangRawTemplateLiteral createRawTemplateLiteral(NodeList<Node> members,
         return literal;
     }
 
-    private BLangNode createRegExpTemplateLiteral(NodeList<Node> reSequences, Location location) {
+    private BLangNode createRegExpTemplateLiteral(NodeList<Node> reSequences, Location location, Token startBackTick,
+                                                  Token endBackTick) {
         BLangRegExpTemplateLiteral regExpTemplateLiteral =
                 (BLangRegExpTemplateLiteral) TreeBuilder.createRegExpTemplateLiteralNode();
-        regExpTemplateLiteral.reDisjunction = (BLangReDisjunction) createReDisjunctionNode(reSequences, location);
+        
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/39466/commits/1d6a475506892f8791a542f5b35d14d606218d7f.